native_client/deepspeech.cc

```diff
@@ -320,55 +320,24 @@ ModelState::infer(const float* aMfcc, unsigned int n_frames, vector<float>& logi
 char*
 ModelState::decode(vector<float>& logits)
 {
-  const int top_paths = 1;
+  const int cutoff_top_n = 40;
```

Comment:
Why this magic number?

Reply:
This is the just the default value used in ctcdecode.